features/step_definitions/machine.rb

```diff
@@ -62,16 +55,23 @@
   cb["new_node"] = node_name
 end
 
-Then(/^admin ensures node number is restored to #{QUOTED} after scenario$/) do | num_expected |
+Then(/^admin ensures machine number is restored to #{QUOTED} after scenario$/) do | num_machine |
```

Comment:
Can we rename the step to `admin checks machine number is restored after scenario`.
Then the step will retrieve machine number from API and check that number is restored after scenario. I mean any reason we should specify number of machines when step can obtain that number very easily by itselt?

Reply:
Yes, I renamed it.